Lib/test/test_codecs.py

```diff
@@ -3107,6 +3108,13 @@ def test_aliases(self):
                     info = codecs.lookup(alias)
                     self.assertEqual(info.name, expected_name)
 
+    def test_alias_modules_exist(self):
+        encodings_dir = os.path.dirname(encodings.__file__)
+        for value in encodings.aliases.aliases.values():
+            codec_file = os.path.join(encodings_dir, value + ".py")
+            self.assertTrue(os.path.isfile(codec_file),
+                            "Codec file not found: " + codec_file)
```

Comment:
A follow-up for this test: https://github.com/python/cpython/pull/134777

We see failures when running this in Fedora with installed Python 3.14.0b2.

Reply:
Thanks. I'll have a look.